core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -31,13 +31,13 @@ import org.apache.kafka.common.protocol.{ApiKeys, Errors}
 import org.apache.kafka.common.requests._
 import org.apache.kafka.common.security.auth.KafkaPrincipal
 import org.apache.kafka.common.utils.{Sanitizer, Time}
-import org.apache.log4j.Logger
+import org.slf4j.LoggerFactory
 
 import scala.collection.mutable
 import scala.reflect.ClassTag
 
 object RequestChannel extends Logging {
-  private val requestLogger = Logger.getLogger("kafka.request.logger")
+  private val requestLogger = LoggerFactory.getLogger("kafka.request.logger")
```

Comment:
It seems like we are using `slf4j` instead of `scala-logging` in a number of places. Is there a reason for that?

Reply:
Noticed that yesterday when I resolved the conflict, the fix is on the way. It should be scala-logging.